static/js/base.js

```diff
@@ -131,6 +133,38 @@ function runAutoTimer() {
     }, 1000);
 }
 
+function setUpCanvas() {
+  canvas = document.getElementById('fieldCanvas')
+  ctx = canvas.getContext('2d')
+  setUpWalls(ctx);
+
+
+  // outlined square X: 50, Y: 35, width/height 50
+  //ctx.beginPath()
+  //ctx.strokeRect(50, 35, 50, 50)
+
+  // filled square X: 125, Y: 35, width/height 50
+  //ctx.beginPath()
+  //ctx.fillRect(125, 35, 50, 50)
+}
+
+function setUpWalls(ctx) {
+    let wallNum = 4; //change this if you want
+    let arr = new Array([0, 0, 144, 2], [0, 0, 2, 144], [142, 0, 2, 144], [0, 142, 144, 2]);
```

Comment:
Maybe add a comment saying that changing this to read from a data file is a TODO